web/settings/modulconfigpv.php

```diff
@@ -952,6 +952,7 @@ function display_pvwattmodul() {
 										<option <?php if($pv2wattmodulold == "wr2_solax") echo "selected" ?> value="wr2_solax">Solax</option>
 										<option <?php if($pv2wattmodulold == "wr2_sungrow") echo "selected" ?> value="wr2_sungrow">Sungrow</option>
 										<option <?php if($pv2wattmodulold == "wr2_victron") echo "selected" ?> value="wr2_victron">Victron MPPT</option>
+										<option <?php if($pv2wattmodulold == "wr2_solarlog") echo "selected" ?> value="wr2_solarlog">SolarLog</option>
```

Comment:
Bitte den neuen Eintrag alphabetisch einsortieren.

Reply:
Ok, mach ich nach dem WE. Bin gerade auf dem Weg in die Kitzbüheler Alpen :-)